src/main/java/seedu/duke/commands/member/TeamMembersListCommand.java

```diff
@@ -9,24 +9,36 @@
 public class TeamMembersListCommand extends Command {
 
     private String name;
+    private boolean isHomeView;
+    private int projectIndex;
 
-    public TeamMembersListCommand() {
+    public TeamMembersListCommand(boolean isHomeView, int projectIndex) {
+        this.isHomeView = isHomeView;
```

Comment:
We have not abstracted the HomeView for other commands so maybe leave the HomeView in the Parser?